### ISSUE TITLE
5.2.3.0 doesn´t change the default windows mouse behaviour, whereas 5.2.2.0 does

### ISSUE THREAD

**Issue by LukeFromAfar** [mgth/LittleBigMouse#383]
I used 5.2.3.0, but the mouse on multiple monitors did not work. When i downgraded as other users suggested, to 5.2.2.0, it worked.

**Comment by JillySOL** [comment]
Correct, 5.2.3 does not work at all it seems. It freezes when running/stopping.

**Comment by masahmad71** [comment]
in my case, it works at beginning, then stops working after updated the display driver. Version 5.2.2.0 works fine.

**Comment by joelspadin** [comment]
I also tried installing 5.2.3.0 from winget, and the program did not work. Both the start and stop buttons did not do anything.

After uninstalling that and installing 5.2.2.0 from the GitHub releases, it started working correctly.

**Comment by andrazek** [comment]
Same here.

**Comment by mgth** [comment]
👋 **[LittleBigMouse 5.3.0-beta.1](https://github.com/mgth/LittleBigMouse/releases/tag/v5.3.0-beta.1) is out and should fix this.**

Several bugs could make the app silently do nothing and are fixed in this release:

- a startup crash when a monitor exposes no EDID (Remote Desktop, virtual displays, DisplayLink, spacedesk) — the app showed no window at all
- the mouse daemon silently failed to launch (broken path resolution)
- the process exclusion file was mistakenly created as a *directory*, which also aborted the daemon launch (the beta self-heals existing installs)

⚠️ The beta requires the [.NET 10 Desktop Runtime (x64)](https://dotnet.microsoft.com/en-us/download/dotnet/10.0) — if the UI still doesn't appear, install that first.

Could you give it a try and report back? If the problem is gone, this issue will be closed with the stable release.

**Comment by mgth** [comment]
No news since the re-test request two weeks ago — closing as fixed by the 5.3.0/5.4.0 engine fixes, as announced. If you still see this on [5.4.1](https://github.com/mgth/LittleBigMouse/releases/tag/v5.4.1), just leave a comment and it reopens.
